plugins/language_conf.lua

```diff
@@ -0,0 +1,46 @@
+-- mod-version:3
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "Conf",
+  files = { ".gitconfig", ".rclone.conf", "%.conf$" },
```

Comment:
Actually in the case of `.gitconfig` you sometimes want to have multiple `.gitconfig*` files so that you can use different 'personas' depending on the project, e.g.:

`.gitconfig`
```
[user]
    email = mrhackerman@aol.com
    signingkey = 123
[includeIf "gitdir:~/WorkDir/"]
    path = .gitconfig-work
```

`.gitconfig-work`
```
[user]
    email = johndoe@ibm.com
    signingkey = 456
```